### PR TITLE
fixed typos and adjusted documentation to reflect the Github document…

### PR DESCRIPTION
**Pull request by piecedigital**
…ation

Let me know if there are any issues with this PR.

### CODE REVIEW

**Comment by piecedigital** [comment]
#183

**Comment by designfrontier** [comment]
This is looking great! Just ran through it really quickly will take a better look tonight. But it's looking fantastic.

File: index.html [hunk 1]
```diff
@@ -100,11 +100,21 @@
 <pre><code>
   events.on('route:/profile:get', function (connection) {
     connection.res.end('route /profile now responding to get requests');
+    // If those more familiar with using ".send()" when finishing
+    // eventhandlers it is also available as an alternative,
+    // like this:
+    // connection.res.send('route /profile now responding to get requests');
```
Comment (designfrontier) [inline index.html]: probably want to prefer `.send` over `.end` in the docs. It provides a better developer experience and less work when implementing sites.

**Comment by designfrontier** [comment]
Other then that one note this looks fantastic @piecedigital! I don't know if we are favoring `.send` elsewhere in the docs but we should. The original docs were written before it was introduced to the codebase. :-/ If we aren't preferring it in the docs you were looking at could you throw an issue in to get that corrected as well?

Thanks again!

**Pull request merged**